queries_go18_test.go

```diff
@@ -264,3 +264,57 @@ func TestContext(t *testing.T) {
 		return
 	}
 }
+
+func TestNamedParameters(t *testing.T) {
+	conn := open(t)
+	defer conn.Close()
+	row := conn.QueryRow(
+		"select :param2, :param1, :param2",
+		sql.Param("param1", 1),
+		sql.Param("param2", 2))
+	var col1, col2, col3 int64
+	err := row.Scan(&col1, &col2, &col3)
+	if err != nil {
+		t.Errorf("Scan failed with unexpected error %s", err)
+		return
+	}
+	if col1 != 2 || col2 != 1 || col3 != 2 {
+		t.Errorf("Unexpected values returned col1=%d, col2=%d, col3=%d", col1, col2, col3)
+	}
+}
+
+func TestBadNamedParameters(t *testing.T) {
+	conn := open(t)
+	defer conn.Close()
+	row := conn.QueryRow(
+		"select :param2, :param1, :param2",
+		sql.Param("badparam1", 1),
```

Comment:
I see, I saw this it is Named in doc but my version of go had Param instead.  Will update my go version and push again.